components/ui/data-table.tsx

```diff
@@ -61,6 +63,19 @@ export function DataTable<TData, TValue>({
     },
   });
 
+  const [selectAll, setSelectAll] = useState(false);
```

Comment:
initial state should be true as all columns are selected. 
![image](https://github.com/PathoTracker/Pathotracker/assets/29795835/a6404796-b137-4734-b5e0-d3e7a1791f3f)

Reply:
Also the state should update when something is unselected manually:

![image](https://github.com/PathoTracker/Pathotracker/assets/29795835/401eb8e2-cd17-4027-9533-021457688d95)